source/cmake_modules/CompilationFlags.cmake

```diff
@@ -24,9 +24,9 @@ endif(NOT SKIP_TESTS)
 # Gcc Release flags
 if(CMAKE_COMPILER_IS_GNUCXX)
     set(CMAKE_GNUCXX_WARN_FLAGS "-Wall -Wextra -Woverloaded-virtual -Wundef -pedantic")
-    set(CMAKE_GNUCXX_COMMON_FLAGS "-std=c++14 -rdynamic -g -fno-omit-frame-pointer")
+    set(CMAKE_GNUCXX_COMMON_FLAGS "-std=c++14 -rdynamic -fno-omit-frame-pointer")
```

Comment:
This `-g` is for adding debug symbols right ? 
If so I think it should be kept here, as we rely on having debug symbols in release builds

Reply:
Fixed !